### PR TITLE
Initial version for internationalization

### PR DESCRIPTION
**Pull request by HuifangYeo**

### CODE REVIEW

File: notebooks/02-technical-tutorials/internationalization/main.ipynb [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "155cd64b-4838-423a-99c9-65b67e21ed3b",
+   "metadata": {},
+   "source": [
+    "## Translations  \n",
+    "\n",
+    "For internationalization to work, we will need the respectively translation file saved as json, named according to its [locale name](https://www.ibm.com/docs/en/radfws/9.6.1?topic=overview-locales-code-pages-supported), e.g. for French, we have `fr_FR.json`, and for Chinese, we can have `zh-CN.json` or `zh-TW.json`.  \n",
```
Comment (printhellohetal) [inline notebooks/02-technical-tutorials/internationalization/main.ipynb:46]:
```suggestion
    "For internationalization to work, we will need the respective translation file saved as a json, named according to its [locale name](https://www.ibm.com/docs/en/radfws/9.6.1?topic=overview-locales-code-pages-supported), e.g. for French, we have `fr_FR.json`, and for Chinese, we can have `zh-CN.json` or `zh-TW.json`.  \n",
```

**Review by printhellohetal — APPROVED** [review]

File: notebooks/02-technical-tutorials/internationalization/main.ipynb [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "200d48a1-0c0f-4826-bd6f-9652acd6fe1f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Deleting the .ipynb_checkpoints folder in the translations folder as temporary workarounds\n",
+    "# https://github.com/atoti/atoti/issues/745\n",
```
Comment (rcolle) [inline notebooks/02-technical-tutorials/internationalization/main.ipynb:84]: It looks like the associated PR has been merged: https://activeviam.atlassian.net/browse/PIVOT-6640
I've asked to follow up on this.

**Comment by rcolle** [comment]
Looks OK to me. Any particular item you'd want R&D to check?

File: notebooks/02-technical-tutorials/internationalization/main.ipynb (hunk 2 above)
Comment (HuifangYeo) [reply notebooks/02-technical-tutorials/internationalization/main.ipynb:84]: I've tested with v0.7.4 and it's still not working. So I suppose it's not merged.

**Pull request merged**